packages/nimble-components/src/rich-text/editor/tests/rich-text-editor-mention.spec.ts

```diff
@@ -1242,7 +1242,8 @@ describe('RichTextEditorMentionListbox', () => {
             expect(pageObject.isMentionListboxOpened()).toBeFalse();
         });
 
-        it('should filter and commit first mention into the editor on Enter', async () => {
+        // Intermittent, see https://github.com/ni/nimble/issues/2150
```

Comment:
@jattasNI are you planning on disabling the other intermittent test that had failed?

Reply:
Yep, just lost track of it yesterday. I've now disabled the 2nd test and updated the bug to track it too.